sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -189,7 +189,7 @@ def __init__(
             ]
 
         transport = kwargs.pop("transport", None)
-        self.pipeline_client = PipelineClient(url_connection, "empty-config", transport=transport, policies=policies)
+        self.pipeline_client = PipelineClient(base_url=url_connection, config="empty-config", transport=transport, policies=policies)
```

Comment:
You can remove the config parameter entirely - we were only passing something in because it was a required parameter :)